pkgs/applications/version-management/sourcehut/update.sh

```diff
@@ -36,19 +38,33 @@ update_version() {
 
   (cd "$root" && update-source-version "sourcehut.python.pkgs.$1" "$version")
 
+  # Update vendorSha256 of Go modules
+  nixFile="${1%srht}".nix
+  nixFile="${nixFile/build/builds}"
+  retry=true
+  while "$retry"; do
+    retry=false;
+    exec < <(exec nix -L build -f "$root" sourcehut.python.pkgs."$1" 2>&1)
```

Comment:
This only works on unstable.

Reply:
@SuperSandro2000, could you elaborate please? What is failing? And does it have to be fixed?

Reply:
nix build is a experimental feature behind the nix-command flag and does not work by default IIRC.